resources/europe/hungary/hu-facebook-group.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "hu-facebook",
+  "type": "facebook",
+  "account": "osm.hu",
+  "locationSet": {"include": ["hu"]},
+  "languageCodes": ["hu"],
+  "order": 3,
+  "strings": {
+    "community": "OpenStreetMap Hungary FB Group",
```

Comment:
This is also already on the troubleshooting list; half of the time they come with a regular facebook page, where it's somewhat redundant for people to understand it's facebook. As I said in the first post, it would be fine to write them out fully, though it may clutter the list slightly because of overflow into another line.

Reply:
What troubleshooting list do you mean?

Reply:
The one [in the first post](https://github.com/osmlab/osm-community-index/pull/524#issue-1190321619).

Reply:
Resolved in https://github.com/danieldegroot2/osm-community-index/commit/33079cf7be900e102f7cef4d877f5241c3703c76 and other commits.